src/app/child-dev-project/attendance/model/activity-attendance.ts

```diff
@@ -101,37 +93,51 @@ export class ActivityAttendance extends Entity {
     return this.countIndividual(childId, AttendanceLogicalStatus.ABSENT);
   }
 
+  countEventsUnknown(childId: string): number {
+    return this.countIndividual(childId, AttendanceLogicalStatus.IGNORE);
```

Comment:
`AttendanceLogicalStatus.IGNORE` and `currentEvent.hasUnknownAttendances()` are not the same thing!

the latter tests whether an event has participants for which any information might be missing in the object, while explicitly set status to "ignore" (e.g. for excused participants) is okay at not an "unknown" status.

Reply:
Then we have a inconsistency with the behavior for a specific child where the "unknown status" is the number of events where the status it `AttendanceLogicalStatus.IGNORE`.

Reply:
Makes sense. Not too happy with the unknown status for recurring activity yet as this is showing a total of events while the other totals are showing a total of participants but lets leave this for a later restructuring.

Reply:
good point. One more issue for the redesign of attendance: #1184